### PR TITLE
Added recipe for shutilwhich

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/shutilwhich/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - python
+
+test:
+  imports:
+    - shutilwhich
+
+about:
+  home: http://github.com/mbr/shutilwhich
+  license: PSF
+  # license_file: No LICENSE or MANIFEST.in
```
Comment (jakirkham) [inline recipes/shutilwhich/meta.yaml]: Can we please raise an issue/PR upstream and xref it here?

File: recipes/shutilwhich/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - shutilwhich
+
+about:
+  home: http://github.com/mbr/shutilwhich
+  license: PSF
```
Comment (jakirkham) [inline recipes/shutilwhich/meta.yaml]: Which version?

**Comment by jakirkham** [comment]
Any thoughts on the comments above, @pmlandwehr?

File: recipes/shutilwhich/meta.yaml (hunk 2 above)
Comment (pmlandwehr) [inline recipes/shutilwhich/meta.yaml]: See https://github.com/mbr/shutilwhich/issues/2

**Comment by jakirkham** [comment]
Thanks @pmlandwehr . Looks good. 👍

**Pull request merged**